openassessment/locale/en@lolcat/LC_MESSAGES/djangojs.po

```diff
@@ -1,297 +0,0 @@
-# edX translation file.
```

Comment:
Do we know what `en@lolcat` is?

Reply:
From my understanding it seems a temp directory which is no longer used because i was not able to find the language page on transifex. and its not been updated in a long time so we can safely remove those.

Reply:
ok